packages/wallet/src/redux/protocols/direct-funding/reducer.ts

```diff
@@ -127,6 +137,12 @@ const fundingReceiveEventReducer: DFReducer = (
 ): ProtocolStateWithSharedData<states.DirectFundingState> => {
   // TODO[Channel state side effect]: update funding level for the channel.
 
+  if (!protocolState.exchangePostFundSetups) {
+    return {
+      protocolState: states.fundingSuccess(protocolState),
```

Comment:
This seems wrong to me, though perhaps I don't understand what `exchangePostFundSetups` means.